src/tokens.rs

```diff
@@ -214,4 +214,31 @@ impl ColorPreset {
             Self::Custom([r, g, b]) => [r, g, b],
         }
     }
+    pub(crate) const fn label(self) -> &'static str {
```

Comment:
I added this so I can make `Custom` label in the dropdown plain, without messing with the `Debug` impl.

Reply:
Can we replace DROPDOWN_TEXT with `self.themes[i].label()`?

Reply:
> Can we replace DROPDOWN_TEXT with `self.themes[i].label()`?

Done

Reply:
On 2nd thought: leave it as it was.

Reply:
> On 2nd thought: leave it as it was.

Can you explain? Do you want me to restore the `DROPDOWN_TEXT` array?
What's the rationale behind that?